scripts/index.ts

```diff
@@ -16,7 +16,7 @@ async function run(steps: string[]) {
         'npm',
         [
           'run',
-          ...moduleAndArgs.slice(1)
+          moduleAndArgs.slice(1).join('-')
```

Comment:
This seems like a logical rather than style change. Intentional?

Reply:
This is a build script -- wanted to support npm commands that have a hyphen in its name.